test/__snapshots__/sqlite.test.ts.snap

```diff
@@ -0,0 +1,615 @@
+      "lineno": 4,
+      "method_id": "main",
+      "parameters": [],
+      "path": "./index.js",
+      "static": true,
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 2,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run'",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 3,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run'",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 4,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run with callback'",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 5,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run with callback'",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 6,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run with params', ?, ?",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 7,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run with params', ?, ?",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 8,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run with params and callback', ?, ?",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 9,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run with params and callback', ?, ?",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 10,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run with object param', $foo, $bar",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 11,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.run with object param', $foo, $bar",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 12,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.all'",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 13,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.all'",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 14,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.all with callback'",
+      },
+      "thread_id": 0,
+    },
+    {
+      "event": "call",
+      "id": 15,
+      "sql_query": {
+        "database_type": "sqlite",
+        "sql": "SELECT 'Database.all with callback'",
+      },
+      "thread_id": 0,
+    },
```

Comment:
All these are duplicated. As far as I can tell this is because `Database` calls (except `exec`) just delegate to `Statement`, so you're recording them twice. If I'm correct you can simplify the code and just hook the `Statement` methods.